src/toast/styled-components.js

```diff
@@ -149,5 +149,6 @@ export const CloseIconSvg = styled<SharedStylePropsArgT>('svg', props => {
     cursor: 'pointer',
     width: '16px',
     flexShrink: 0,
+    paddingTop: '2px',
```

Comment:
arbitrary padding feels a bit restrictive, consider the situation if one overrides with a larger icon. does centering vertically work well?

Reply:
Yeah, that's a good point. I don't think there's a good way to do this scalably.

If we're choosing between vertically centering and keeping it as is, I think I'd rather do the latter.

Reply:
sounds good - I'll close this, and let's leave things as they are